contents/blog/data-management-feature.md

```diff
@@ -0,0 +1,83 @@
+---
+date: 2022-03-23
+title: You can now organize your PostHog data
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+author: ["alex-kim"]
+categories: ["Product updates"]
+---
+
+As PostHog grows, the data that users are sending to PostHog continues to grow into the billions.
+
```

Comment:
Maybe some examples with numbers? e.g. certain customer who has ~30k unique events. Another approach could be to show a before/after picture?

Reply:
Nice, I wasn't sure if we could show data from prod here but I guess just the counts are fine.

Reply:
@jamesefhawkins are you happy with these numbers being public